system/jlib/jstring.cpp

```diff
@@ -216,13 +216,15 @@ StringBuffer & StringBuffer::append(const char * value)
     return *this;
 }
 
-StringBuffer & StringBuffer::append(unsigned len, const char * value)
+StringBuffer & StringBuffer::append(size_t len, const char * value)
```

Comment:
Changed this so that ptr1-ptr2 can be passed without issues of truncation.  I'll open a jira to switch the whole class over to using size_t.